src/file/index.ts

```diff
@@ -5,4 +5,3 @@ export * from "./numbering";
 export * from "./media";
 export * from "./drawing";
 export * from "./styles";
-export * from "./xml-components";
```

Comment:
If we don't export `XmlComponent` then there is no easy option for the users of the library to easy create new components. For example I would like to write:
```
class SomeOtherField extends XmlComponent {
}
```
that might have some logic that is specific for my usage, but not relevant for main library.

Or there is different way how can we can achieve this ?

Reply:
I see makes sense. I have never thought of that use case!

My initial thoughts were that it breaks encapculation, but your reasoning makes perfect sense

Will add back in